tensorflow/python/keras/losses.py

```diff
@@ -38,6 +38,8 @@
 from tensorflow.python.ops.losses import util as tf_losses_util
 from tensorflow.python.util.tf_export import keras_export
 from tensorflow.tools.docs import doc_controls
+from tensorflow.math import reduce_sum, square
+from tensorflow import size, cast, float32
```

Comment:
Note that you will import here considering tensorflow codebase and not API.  
In my opinion, these imports won't work. You need to import considering codebase here similar to other imports in the file.

Reply:
Ah yes. That was terrible from my part. I will change that.